persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -349,6 +351,9 @@ toParsedEntityDef lwc = ParsedEntityDef
     (attribs, extras) =
         parseEntityFields fieldLines
 
+    schemaName =
+      fmap SchemaNameDB $ listToMaybe $ mapMaybe (T.stripPrefix "schema=") entAttribs
```

Comment:
Perhaps we should error if multiple schemas are defined? I haven't done any tests but it appears that specifying the sql name has similar logic (`getDbName`).

Reply:
I think it's fine to leave it like this if it's consistent with the rest of the module.